pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -222,7 +228,24 @@ public void getPendingAckInternalStats(@Suspended final AsyncResponse asyncRespo
                                            @PathParam("topic") @Encoded String encodedTopic,
                                            @PathParam("subName") String subName,
                                            @QueryParam("metadata") @DefaultValue("false") boolean metadata) {
-        internalGetPendingAckInternalStats(asyncResponse, authoritative,
-                TopicName.get(TopicDomain.persistent.value(), tenant, namespace, encodedTopic), subName, metadata);
+
+        internalGetPendingAckInternalStats(authoritative,
+                    TopicName.get(TopicDomain.persistent.value(), tenant, namespace, encodedTopic), subName, metadata)
```

Comment:
I think we need to catch ``TopicName.get(TopicDomain.persistent.value(), tenant, namespace, encodedTopic)`` to avoid ``IllegalArgumentsException`` exception.

Reply:
yes, right. Thanks for this suggestion. The topic name isn't decoded...

Reply:
fixed.